### ISSUE TITLE
[BUG] UI not working

### ISSUE THREAD

**Issue by AleXSR700** [flixlix/power-flow-card-plus#188]
**Describe the bug**
When trying to create a card, the following error is shown:
<img width="739" alt="image" src="https://github.com/flixlix/power-flow-card-plus/assets/57034871/2a81b798-597b-4f04-aefc-879d6f1c6289">

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Please make sure all of this applies and check the checkboxes, by writing "X" inside of them.**

- [ x] I understand that this is a channel for reporting bugs, not a support forum (If you need help configuring the card, go to: [Discussions](https://github.com/flixlix/power-flow-card-plus/discussions)).

- [x ] I have the latest version of the card installed.

- [x ] I have cleared my browser cache **after** installing, updating and configuring the card.

**Comment by flixlix** [comment]
Known issue. Duplicate of #158

**Comment by AleXSR700** [comment]
Ah, too bad. Why did you close the other issue? I only looked for open ones. It is not resolved, so should it not be open?

Unfortunately, card-mod is needed urgently due to the poor frontend of HA :(

**Comment by flixlix** [comment]
I closed the issues because they are not caused by this card but instead card mod is causing the problems. There is nothing that can be done about this card to change that.

I had already added this warning in the readme to avoid these issues.

And also, as I said in the readme, I did open a PR in card mod fixing the issue, but as long as that is open you unfortunately won't be able to use the UI editor of this card when having card mod installed.
